css/main.css

```diff
@@ -0,0 +1,54 @@
+body {
+	background: black;
+	color: white;
+	display: flex;
+	justify-content: center;
+	background-image: url(http://i.kinja-img.com/gawker-media/image/upload/msykmohxox4buoeuvp2y.jpg); 
```

Comment:
Nice use of flexbox. Also, I wanna say that the laser is not coming from the cat in that picture.